RecoLocalCalo/EcalRecProducers/python/ecalRecHitGPU_cfi.py

```diff
@@ -0,0 +1,132 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoLocalCalo.EcalRecAlgos.ecalCleaningAlgo import cleaningAlgoConfig 
+
+# rechit producer
+ecalRecHitGPU = cms.EDProducer("EcalRecHitProducerGPU",
+                               
+    uncalibrecHitsInLabelEB = cms.InputTag("ecalUncalibRecHitProducerGPU","EcalUncalibRecHitsEB"),
+    uncalibrecHitsInLabelEE = cms.InputTag("ecalUncalibRecHitProducerGPU","EcalUncalibRecHitsEE"),
+          
+    #recHitsLabelEB = cms.string("EcalRecHitsGPUEB"),
+    #recHitsLabelEE = cms.string("EcalRecHitsGPUEE"),
+    recHitsLabelEB = cms.string("EcalRecHitsEB"),
+    recHitsLabelEE = cms.string("EcalRecHitsEE"),
+ 
+    maxNumberHits = cms.uint32(20000),  # FIXME AM
+  
+  
+    #EErechitCollection = cms.string('EcalRecHitsEE'),
+    #EEuncalibRecHitCollection = cms.InputTag("ecalMultiFitUncalibRecHit","EcalUncalibRecHitsEE"),
+    #EBuncalibRecHitCollection = cms.InputTag("ecalMultiFitUncalibRecHit","EcalUncalibRecHitsEB"),
+    #EBrechitCollection = cms.string('EcalRecHitsEB'),
+   
+    ## db statuses to be exluded from reconstruction (some will be recovered)
+    ChannelStatusToBeExcluded = cms.vstring(   'kDAC',
+                                               'kNoisy',
+                                               'kNNoisy',
+                                               'kFixedG6',
+                                               'kFixedG1',
+                                               'kFixedG0',
+                                               'kNonRespondingIsolated',
+                                               'kDeadVFE',
+                                               'kDeadFE',
+                                               'kNoDataNoTP',
+                                               #
+                                               # AM should I add them here?????
```

Comment:
who would know ?

Reply:
this is part of "how to improve and make it simpler" for cpu and gpu :)